suite-native/assets/src/assetsSelectors.ts

```diff
@@ -58,6 +59,12 @@ export const selectDeviceAssetsWithBalances = memoize(
         const fiatCurrencyCode = selectFiatCurrencyCode(state);
 
         return deviceNetworksWithAssets
+            .sort((a, b) => {
```

Comment:
Sort of dangerous because it mutates original object, use A.sort or toSorted

Reply:
2513f11206